mosdef_cassandra/writers/inp_functions.py

```diff
@@ -236,37 +240,44 @@ def generate_input(system, moves, run_type, run_length, temperature, **kwargs):
     boxes = []
     for box in system.boxes:
         if isinstance(box, mbuild.Compound):
-            box_dims = np.hstack((box.periodicity, box.boundingbox.angles))
```

Comment:
What am I missing with the change here?

Reply:
This was something I initially changed and need to change back.